src/lucky/secure_headers/set_xss_guard.cr

```diff
@@ -0,0 +1,33 @@
+module Lucky
+  module SecureHeaders
+    # This module sets the HTTP header [X-XSS-Protection](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection).
+    # It's job is responsible for telling the browser to not render a page if it detects cross-site scripting.
+    # For Internet Explorer browsers less than 9, it's recommended to disabled it. Read more on [Microsoft](https://blogs.msdn.microsoft.com/ieinternals/2011/01/31/controlling-the-xss-filter/).
```

Comment:
I think it might be good to tell people that lucky implements this recommendation so they don't think they have to write code to do so. What do you think?

Reply:
I think what Edward meant was that this comment makes it sound like you have to disable. It seems like Lucky isn't doing it for you. 

So maybe something like:

`Lucky disables this header for Internet Explorer 9 because it is recommended to do so. Read more on...`

Reply:
Ah yes, sorry about that. Poor communication on my part, thanks @paulcsmith. It does seem like the framework is instructing developers to opt out of this behavior in IE9 and below, when in fact the framework is doing that work for the developer.

Reply:
ah! Gotcha. Ok, that makes sense. My bad 😆